packages/amplify-e2e-core/src/categories/notifications.ts

```diff
@@ -43,7 +43,7 @@ export const addNotificationChannel = async (
   channel: string,
   hasAnalytics = false,
   hasAuth = false,
-  testingWithLatestCodebase = true,
+  testingWithLatestCodebase = false,
```

Comment:
`false` seems correct (it's a pattern everywhere).
was `true` introduced by mistake? or was it always here?

Reply:
It must have been introduced by mistake, our windows image on circleci was forgiving & allowed this error to go unnoticed.